source/Octopus.Client/Repositories/DashboardRepository.cs

```diff
@@ -6,9 +6,9 @@ namespace Octopus.Client.Repositories
     public interface IDashboardRepository
     {
         DashboardResource GetDashboard();
-        DashboardResource GetDynamicDashboard(string[] projects, string[] environments);
+        DashboardResource GetDynamicDashboard(string[] projects, string[] environments, bool includePrevious = false);
```

Comment:
What does this param actually do? As a caller it isn't clear to me what previous thing is actually going to get included here if I pass `true`.

Reply:
Ah so right now, the DashboardResource has an `items` property which includes a list of releases, However as it is right now, it only ever gets you 1 item which is the latest release. The dashboard route however, does let you set this to true, and therefore returns all the previous releases that were successful. We need this in OctopusCLI in order to promote the most recent successful release.

But yes! it's not really clear. Perhaps instead of a straight boolean, 
So I guess two options - 

1) add documentation to that method, explaining what the parameter is for

2) add DynamicDashboardOptions type, which has more descriptive `includePreviousReleases` property? 

```
DashboardResource GetDynamicDashboard(string[] projects, string[] environments, DynamicDashboardOptions options);

...
GetDynamicDashboard(projects, environments, {includePreviousOptions:true})

```

Reply:
Even wrapped up in a type, what does `includePreviousReleases:true` mean?